### DISCUSSION TITLE
Main menu buttons

### DISCUSSION THREAD

**Discussion by AltronMaxX** [KadeArchive/Kade-Engine#2461] [Q&A]
How con I edit X position of buttons (story mode, freeplay) of main menu? I'm using Kade-Engine 1.7

**Comment by SanicBTW** [comment]
I used to go to the Alphabet.hx file and modify the x on the ismenuitem or something, I don't remember but if you want to center the main menu items it should be somewhere around MainMenuState.hx, sorry I've been using Psych Engine 0.3.2 for some time already and I only used KE 1.6.x for HTML5 builds

**Reply by AltronMaxX** [reply]
I have already found solution for my problem, but thanks
